cmd/v/help/install.txt

```diff
@@ -1,13 +1,13 @@
 Usage:
-  v install [MODULE...]
+  v install [GIT_REPO_URL...|MODULE...]
     Installs each MODULE.
 
     If no MODULEs, the modules listed in the `v.mod` file are installed instead.
 
 Options:
-  --vpm         - [Default] Install from vpm
-  --git         - Install from git repository url
-  --hg          - Install from mercurial repository url
+  -git          - Install from git repository url
+  -hg           - Install from mercurial repository url
+  -f|-force     - force to replace module when installing from repositorie's url
```

Comment:
Perhaps just `force replace the module` ?
@JalonSolov what do you think?

Reply:
`force module installation, regardless of existing install`

Reply:
> `force module installation, regardless of existing install`

ok. i will remove -force flag

Reply:
We weren't suggesting removing the option, just changing the usage message.

Reply:
no problem. i will undo the flag